src/components/Greetings.js

```diff
@@ -0,0 +1,18 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { randomGreeting } from '../redux/greeting';
+
+const Greetings = () => {
+  const greetings = useSelector((state) => state.greeting);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(randomGreeting());
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <h1>{greetings.greeting}</h1>
+  );
+};
```

Comment:
According to the project requirements, you are suppose to display the random greetings in your greeting component at the frontend. However, it shows null. Kindly check and fix it to make sure that the greetings are displayed as required.